integration/kubernetes/helm/alluxio/templates/alluxio-master.yaml

```diff
@@ -48,7 +48,7 @@ spec:
     name: web
   {{- if $isEmbedded }}
   - port: 19200
-    name: embedded-journal
+    name: ej
```

Comment:
^

Reply:
Sorry overlooked some of them. These have been changed now.